scrapers/TaraTainton.yml

```
@@ -0,0 +1,26 @@
+name: TaraTainton.com
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - www.taratainton.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@id="leftColumn"]/article/h1
+      Details:
+        selector: //div[@id="leftColumn"]/article/p
+        concat: "\n\n"
+      Studio:
+        Name:
+            fixed: Tara Tainton
+      Date: 
+        selector: //div[@id="leftColumn"]/article/div[@class="theDate"]
+        postProcess:
+          - parseDate: 02 January 2006
```

```suggestion
          - parseDate: 2 January 2006
```
from a couple of scenes i tested no zero filling is done, so the date part shouldnt have a zero